### PR TITLE
Pin adm-zip version to prevent breaking changes

### PR DESCRIPTION
**Pull request by thomasheritage**
Pin adm-zip version to avoid fatal errors such as this seen on a recent SMPTE HTML doc repo GitHub Actions workflow:
```
TypeError: val.getFullYear is not a function
    at Utils.fromDate2DOS (/home/runner/work/st2067-206-private/st2067-206-private/node_modules/adm-zip/util/utils.js:328:13)
    at set time [as time] (/home/runner/work/st2067-206-private/st2067-206-private/node_modules/adm-zip/headers/entryHeader.js:101:34)
    at makeLibraryZip (file:///home/runner/work/st2067-206-private/st2067-206-private/tooling/scripts/build.mjs:476:23)
    at main (file:///home/runner/work/st2067-206-private/st2067-206-private/tooling/scripts/build.mjs:774:37)
```
which is caused by:
https://github.com/cthackers/adm-zip/issues/539

(It looks like other breaking changes have been reported with adm-zip in recent releases)

### CODE REVIEW

**Review by palemieux — APPROVED** [review]

**Comment by thomasheritage** [comment]
Thanks @palemieux for approving

Note:
* Build checks failing (error in loading AWS credentials)
* I don't have write access to this repo so can't merge this PR (perhaps I could be granted write access?)

**Comment by thomasheritage** [comment]
Thanks -- merged

**Pull request merged**
